vmdb/db/fixtures/ae_datastore/ManageIQ/Infrastructure/VM/Retirement/StateMachines/Methods.class/__methods__/unregister_from_provider.rb

```diff
@@ -1,10 +0,0 @@
-#
-# Description: This method unregisters the VM from the provider
-#
-
-vm = $evm.root['vm']
-unless vm.nil?
-  ems = vm.ext_management_system
-  $evm.log('info', "Unregistering VM:<#{vm.name}> from EMS:<#{ems ? ems.name : nil}")
-  vm.unregister
-end
```

Comment:
@gmcculloug @tinaafitz 
Since this method is being deleted and being combined into remove_from_provider.rb with an input argument (remove_from_disk or unregister). We don't have an instance to handle the unregister case. The previous model had a UnregisterFromProvider which has been removed in the new model, so if a user would like to use the optional "unregister" he would have to know the the argument to pass into the remove_from_provider method. We should either document this or provide a sample instance that has the correct input parameter akin to the instance RemoveFromProvider.